sycl/test-e2e/Basic/memory-consumption.cpp

```diff
@@ -7,6 +7,8 @@
 // extra memory allocation on device creation.
 // Issue #661: The test is failing sporadically on HIP AMD.
```

Comment:
It seems this test has been already identified "flaky" although it was on HIP AMD platform before.
Anyway, measuring memory consumption at two different times cannot check memory leak reliably.
Let's remove this test from our E2E test suite.

Reply:
done in latest commit